metricbeat/helper/http_test.go

```diff
@@ -0,0 +1,54 @@
+package helper
```

Comment:
I realize now this is a new file without license headers, but CI is not complaining. Are we checking that somewhere @andrewkroh?

Reply:
@exekias Was also looking into this this morning. We check but return 0 after the check. Not sure if that was on purpose (probably not).

Reply:
In the meanwhile, I've pushed the header to this file